Source/WebCore/bindings/js/SerializedScriptValue.cpp

```diff
@@ -2858,10 +2839,17 @@ class CloneDeserializer : CloneBase {
 
     bool read(bool& b)
     {
-        int32_t integer;
-        if (!readLittleEndian(integer) || integer > 1)
-            return false;
-        b = !!integer;
+        if (m_version >= 14) {
```

Comment:
I don’t understand why we need this check. Does cloning actually need compatibility with older versions?

Reply:
As far as I know, we need to preserve backward compatibility. I think it is because such serialization can be stored via IndexedDB. This is the reason we have versioning.